docs/guide/configuration.md

```
@@ -259,6 +259,69 @@ add_filter('c3_log_invalidation_list', '__return_true');
+Debug settings are now managed through **Settings > Reading > C3 CloudFront Debug Settings** in WordPress admin.
+
+### Debug Log Output
+
+When debug settings are enabled, detailed logs are written to your WordPress error log. Check the following locations:
+
+- **WordPress Debug Log**: `wp-content/debug.log`
+- **Server Error Log**: Check your hosting provider's error log location
+- **WP-CLI**: Use `tail -f /var/log/php-fpm/www-error.log` to view recent log entries, or install the WP-CLI logs extension with `wp package install wp-cli/logs-command` to use `wp log list`
+
+Example debug log output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)